src/os/bluestore/BlueFS.cc

```diff
@@ -196,7 +196,7 @@ BlueFS::BlueFS(CephContext* cct)
   discard_cb[BDEV_DB] = db_discard_cb;
   discard_cb[BDEV_SLOW] = slow_discard_cb;
   asok_hook = SocketHook::create(this);
-
+  check_vselector = cct->_conf->bluefs_check_volume_selector_often;
```

Comment:
nit: IMO no much sense to make this a "startup-only" parameter, one can read the config parameter each time it needs to be checked with no penalty.